packages/react-integration/demo-app-ts/package.json

```diff
@@ -18,7 +18,9 @@
     "@types/react-dom": "^16.8.0",
     "@types/react-router-dom": "^4.3.1",
     "react-scripts": "2.1.3",
-    "typescript": "^3.8.3"
+    "typescript": "^3.8.3",
+    "@babel/core": "^7.6.0",
```

Comment:
Remove these deps -- they're already in the root `package.json` and our integration app doesn't need babel.

Reply:
done